cpp/models/abm/testing_strategy.h

```diff
@@ -158,13 +159,13 @@ class TestingScheme
      * checks if the scheme is active at a given time and updates activity status
      * @param t time to be updated at
      */
-    void update_activity_status(const TimePoint t);
+    void update_activity_status(TimePoint t);
 
     /**
      * runs the testing scheme and tests a person if necessary
      * @return if the person is allowed to enter the location
      */
```

Comment:
Add @param for person, location and t here

Reply:
Added.